packages/dicomImageLoader/src/imageLoader/wadors/getPixelData.ts

```diff
@@ -1,100 +1,66 @@
-import { xhrRequest } from '../internal/index';
-import findIndexOfString from './findIndexOfString';
-
-function findBoundary(header: string[]): string {
-  for (let i = 0; i < header.length; i++) {
-    if (header[i].substr(0, 2) === '--') {
-      return header[i];
-    }
-  }
-}
-
-function findContentType(header: string[]): string {
-  for (let i = 0; i < header.length; i++) {
-    if (header[i].substr(0, 13) === 'Content-Type:') {
-      return header[i].substr(13).trim();
-    }
-  }
-}
+import { utilities } from '@cornerstonejs/core';
 
-function uint8ArrayToString(data, offset, length) {
-  offset = offset || 0;
-  length = length || data.length - offset;
-  let str = '';
-
-  for (let i = offset; i < offset + length; i++) {
-    str += String.fromCharCode(data[i]);
-  }
+import { xhrRequest } from '../internal/index';
+// import rangeRequest from '../internal/rangeRequest';
+import streamRequest from '../internal/streamRequest';
+import rangeRequest from '../internal/rangeRequest';
+import extractMultipart from './extractMultipart';
+import { LossyConfiguration } from 'core/src/types';
 
-  return str;
-}
+const { ProgressiveIterator } = utilities;
 
 function getPixelData(
   uri: string,
   imageId: string,
-  mediaType = 'application/octet-stream'
-): Promise<any> {
+  mediaType = 'application/octet-stream',
+  retrieveOptions: LossyConfiguration = {}
```

Comment:
`retrieveOptions: LossyConfiguration` is really unusuall naming to me, is it really about lossy/non-lossy? or it is just about retrieveOptions? Why we can't name the type `RetrieveOptions`?